barretenberg/cpp/src/barretenberg/relations/ultra_arithmetic_relation.hpp

```diff
@@ -83,46 +83,98 @@ template <typename FF_> class UltraArithmeticRelationImpl {
                                   const FF& scaling_factor)
     {
         PROFILE_THIS_NAME("Arithmetic::accumulate");
```

Comment:
some of the later terms in the relation will be of the Accumulator type (e.g. `tmp0` in this file).

I figured as a general rule it might be easier to read what's going on by applying a heuristic that `_m` indicates the type is a MonomialAccumulator, given we use `auto` type declarations for frequently